### ISSUE TITLE
1.9 and older GCE jobs broken due to gcloud update

### ISSUE THREAD

**Issue by shyamjvs** [kubernetes/test-infra#7646]
E.g:
- https://k8s-testgrid.appspot.com/google-gce#gci-gce-scalability-1.7
- https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-1.8
- https://k8s-testgrid.appspot.com/sig-scalability-kubemark#kubemark-small-last-release

The reason is due to gcloud version update done in kubetest that was later updated for all jobs (in https://github.com/kubernetes/test-infra/pull/7635). The newer gcloud version doesn't support `--mode` param for network creation anymore and instead replaced it with `--subnet-mode`:

```
W0411 04:35:58.538] ERROR: (gcloud.compute.networks.create) `mode` has been removed. Please use `subnet-mode` instead.
```

Can't think of a better way out here than cherry-picking PR https://github.com/kubernetes/kubernetes/pull/61203 to 1.9, 1.8 (at least) and if needed 1.7.

cc @rmmh @BenTheElder (who were involved with the above changes)

**Comment by BenTheElder** [comment]
We should cherrypick back the fixes, end users will use newer gcloud as well. We actually just built new images, we don't *try* to pin to old gcloud.

**Comment by shyamjvs** [comment]
This is fixed with cherrypicks to 1.9 and 1.8 (oldest supported release) merged.
/close

**Comment by BenTheElder** [comment]
Thanks! :+1:
